src/systems/geometry.js

```diff
@@ -0,0 +1,133 @@
+var geometries = require('../core/geometry').geometries;
+var registerSystem = require('../core/system').registerSystem;
+var THREE = require('../lib/three');
+
+/**
+ * System for geometry component.
+ * Handle geometry caching.
+ *
+ * @member {object} cache - Mapping of stringified component data to THREE.Geometry objects.
+ * @member {object} cacheCount - Keep track of number of entities using a geometry to
+ *         know whether to dispose on removal.
+ */
+module.exports.System = registerSystem('geometry', {
+  init: function () {
+    this.cache = {};

```

Comment:
Should we merge these two in a single structure of objects ```{ geometry: xxx, count: x }```. We can then retrieve the geometry and increase/decrease the count on a single access.